daisy_workflows/image_import/inspection/src/boot_inspect/cli.py

```diff
@@ -51,22 +55,31 @@ def _output_human(results: model.InspectionResults):
   _output_json(results, indent=4)
 
 
-def _inspect_uefi(device):
+def _inspect_boot_loader(device):
```

Comment:
Since the test framework is ready, move this into one of the subdirectories and include tests for it.

Reply:
I moved _inspect_boot_loader into inspection.py. Would you route me to the test folder? is it inspection/tests?

Reply:
daisy_workflows/image_import/inspection/tests